doc/design/issues/3943 - specify size for bridge networks.md

```diff
@@ -0,0 +1,70 @@
+# Configuration of bridge network widths
+
+## Issues addressed
+
+This design addresses the following issues:
+
+* vmware/vic#3943
+* vmware/vic#3771
+
+The following pull request should be resolved prior to completion of this work:
+
+* vmware/vic#7183
+
+## Requirements
+
+The following requirements should be addressed by this design
+
+1. Configuration of a default width for bridge networks:
+    * In `vic-machine create`
+    * In `vic-machine configure`
```

Comment:
@hickeng if we support to reconfigure the default width, how to make sure all active containers are updated with the updated width?